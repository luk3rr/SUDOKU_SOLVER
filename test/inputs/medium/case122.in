065008300 090070000 000010000 300040002 010009040 900206173 039000408 080000200 047850009 
