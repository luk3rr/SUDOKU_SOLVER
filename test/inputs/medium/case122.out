Solving the following grid:
0 6 5 | 0 0 8 | 3 0 0 
0 9 0 | 0 7 0 | 0 0 0 
0 0 0 | 0 1 0 | 0 0 0 
------+-------+------
3 0 0 | 0 4 0 | 0 0 2 
0 1 0 | 0 0 9 | 0 4 0 
9 0 0 | 2 0 6 | 1 7 3 
------+-------+------
0 3 9 | 0 0 0 | 4 0 8 
0 8 0 | 0 0 0 | 2 0 0 
0 4 7 | 8 5 0 | 0 0 9 


Solution found :')

7 6 5 | 4 2 8 | 3 9 1 
8 9 1 | 6 7 3 | 5 2 4 
4 2 3 | 9 1 5 | 7 8 6 
------+-------+------
3 7 8 | 5 4 1 | 9 6 2 
6 1 2 | 7 3 9 | 8 4 5 
9 5 4 | 2 8 6 | 1 7 3 
------+-------+------
2 3 9 | 1 6 7 | 4 5 8 
5 8 6 | 3 9 4 | 2 1 7 
1 4 7 | 8 5 2 | 6 3 9 

Algorithm: BFS
Total time: 213 ms
Total expanded states: 35176
